src/components/jsmd-editor.jsx

```diff
@@ -61,6 +70,11 @@ class JsmdEditorUnconnected extends React.Component {
     this.props.actions.updateJsmdContent(content);
   }
 
+  updateCursor(editor) {
+    const { line, ch } = editor.getCursor();
+    this.props.actions.updateEditorCursor(line, ch);
```

Comment:
I know this is an older pattern and this file has used `bindActionCreators` for a while, but it might be better to explicitly pick which actions we want in `mapDispatchToProps`.